css/types/color.json

```diff
@@ -196,7 +196,7 @@
                 "version_added": false
               },
               "safari": {
-                "version_added": false
+                "version_added": "TP"
```

Comment:
In the new proposal this is going to be "preview"

Reply:
Just changed it to `"preview"`!